translator/trace/zipkin/zipkinv2_to_traces_test.go

```diff
@@ -128,3 +143,23 @@ func generateTraceSingleSpanMinmalResource() pdata.Traces {
 	rsc.Attributes().UpsertString(conventions.AttributeServiceName, "SoleAttr")
 	return td
 }
+
+func generateTraceSingleSpanErrorStatus() pdata.Traces {
```

Comment:
It's a nit but perhaps [`generateTraceOneSpanOneTraceID`](https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/trace/zipkin/traces_to_zipkinv2_test.go#L109) (or just `testdata.GenerateTraceDataOneSpan`) could be reused to make this even simpler?